docs/user/guides/technical-docs-seo-guide.rst

```diff
@@ -92,30 +81,47 @@ which we'll discuss shortly,
 the most important thing is making sure the spider can follow links on your site
 and get to all your pages.
 
-Avoid orphan pages
-~~~~~~~~~~~~~~~~~~
+Avoid orphan pages ✅️
+~~~~~~~~~~~~~~~~~~~~~~
 
 Sphinx calls pages that don't have links to them "orphans"
```

Comment:
Should we use a different word? `unlinked` seems explicit, and removed the Sphinx-heavy copy.

Reply:
Oh wow, I have overlooked this because of the multitude of Sphinx-only sentences :)

removed :+1: